sessionctx/variable/sysvar.go

```diff
@@ -1954,10 +1957,24 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBOptPrefixIndexSingleScan, Value: BoolToOnOff(DefTiDBOptPrefixIndexSingleScan), Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
 		s.OptPrefixIndexSingleScan = TiDBOptOn(val)
 		return nil
 	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnableReusechunk, Value: BoolToOnOff(DefTiDBEnableReusechunk), Type: TypeBool,
+		SetSession: func(s *SessionVars, val string) error {
+			s.EnableReuseCheck = TiDBOptOn(val)
+			return nil
+		}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBMaxReuseChunk, Value: strconv.Itoa(DefTiDBMaxReuseChunk), Type: TypeUnsigned, MinValue: 1, MaxValue: math.MaxInt32, SetSession: func(s *SessionVars, val string) error {
+		s.MaxReuseChunk = tidbOptPositiveInt32(val, DefTiDBMaxReuseChunk)
+		return nil
+	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBMaxReuseColumn, Value: strconv.Itoa(DefTiDBMaxReuseColumn), Type: TypeUnsigned, MinValue: 1, MaxValue: math.MaxInt32, SetSession: func(s *SessionVars, val string) error {
+		s.MaxReuseColumn = tidbOptPositiveInt32(val, DefTiDBMaxReuseColumn)
+		return nil
+	}},
```

Comment:
The newly introduced variables need to be approved by the PM. IMO, the user may not know how to configure these parameters.